module-apps/application-special-input/ApplicationSpecialInput.cpp

```diff
@@ -9,7 +9,7 @@ using namespace app;
 
 namespace
 {
-    constexpr auto SpecialInputAppStackDepth = 2200;
+    constexpr auto SpecialInputAppStackDepth = 512 * 5;
```

Comment:
tbh, why not full number? :)

Reply:
I think it is clearer, but that's just my opinion.